_layouts/page.html

```diff
@@ -12,9 +12,14 @@ <h1 class="post-title-main">{{ page.title }}</h1>
      {% assign edit_url =  page.url | remove: "/doc" | remove: ".html" | append: ".md" | prepend: site.github_editme_path %}
        <a target="_blank" href="https://github.com/{{edit_url}}" class="btn btn-default githubEditButton" role="button"><i class="fa fa-github fa-lg"></i> Edit this page</a>
    {% endif %}
-   {% if site.lb4_editme_path and page.layout != 'readme' and page.sidebar == 'lb4_sidebar' %}
-     {% assign edit_url =  page.url | remove: "doc/en/lb4" | remove: ".html" | append: ".md" | prepend: site.lb4_editme_path %}
-       <a target="_blank" href="https://github.com/{{edit_url}}" class="btn btn-default githubEditButton" role="button"><i class="fa fa-github fa-lg"></i> Edit this page</a>
+   {% if page.lb4_editme_path %}
```

Comment:
Is there any way to infer the `editme_path` from the md file name? For example:

```
docs/site/decorators/Decorators_authenticate.md 

title: 'Authentication Decorator'
keywords: LoopBack 4.0, LoopBack-Next
sidebar: lb4_sidebar
lb4_editme_path: strongloop/loopback-next/blob/master/docs/site/decorators
permalink: /doc/en/lb4/Decorators_authenticate.html
---
```

If we can access `docs/site/decorators/Decorators_authenticate.md ` as `page.<some_property>`, we don't need to manually configure `lb4_editme_path` per page. At least we can automate this during `update-lb4-docs.js`.